store/db.go

```diff
@@ -91,7 +92,7 @@ func (db *DB) Open() error {
 	if err != nil {
 		unlockErr := db.dl.unlock()
 		if unlockErr != nil {
-			err = fmt.Errorf("[%s, %s]", err, unlockErr)
+			err = errwrap.Wrap(unlockErr, err)
```

Comment:
Yeah, not quite sure how to handle this. I'll think about this a bit for a follow up.